modules/ejbca-ejb-cli/src/org/ejbca/ui/cli/ca/CaImportCertDirCommand.java

```diff
@@ -115,6 +117,9 @@ public CommandResult execute(final ParameterContainer parameters) {
         final String revocationReasonString = parameters.get(REVOCATION_REASON);
         final String revocationTimeString = parameters.get(REVOCATION_TIME);
         final int threadCount = parameters.get(THREAD_COUNT) == null ? 1 : Integer.valueOf(StringUtils.strip(parameters.get(THREAD_COUNT)));
+        final String caCertFile = parameters.get(CACERT);
+        final boolean bRevokeDetailsInFilename = parameters.containsKey(REVOKEDETAILS);
```

Comment:
just call is revokedDetailsInFilename our coding convention does not use any type identifiers. Type is already clear.

Reply:
OK. In fact, this boolean is only used the once, so might not really required.